src/USER-MISC/fix_bond_react.cpp

```diff
@@ -1650,6 +1653,18 @@ void FixBondReact::find_landlocked_atoms(int myrxn)
     }
   }
 
+  // additionally, if a deleted atom is bonded to a undeleted atom, bad
```

Comment:
see my previous inquiry about what is meant by `undeleted` in this context. if it needs to be changed in the docs, it should be changed here, too.